app/uk/gov/hmrc/transitmovementsvalidator/models/errors/ErrorCode.scala

```diff
@@ -27,11 +27,12 @@ import play.api.libs.json.Writes
 /** Common error codes documented in [[https://developer.service.hmrc.gov.uk/api-documentation/docs/reference-guide#errors Developer Hub Reference Guide]]
   */
 object ErrorCode {
-  val BadRequest: ErrorCode           = ErrorCode("BAD_REQUEST", BAD_REQUEST)
-  val NotFound: ErrorCode             = ErrorCode("NOT_FOUND", NOT_FOUND)
-  val InternalServerError: ErrorCode  = ErrorCode("INTERNAL_SERVER_ERROR", INTERNAL_SERVER_ERROR)
-  val SchemaValidation: ErrorCode     = ErrorCode("SCHEMA_VALIDATION", OK)
-  val UnsupportedMediaType: ErrorCode = ErrorCode("UNSUPPORTED_MEDIA_TYPE", UNSUPPORTED_MEDIA_TYPE)
+  val BadRequest: ErrorCode              = ErrorCode("BAD_REQUEST", BAD_REQUEST)
+  val NotFound: ErrorCode                = ErrorCode("NOT_FOUND", NOT_FOUND)
+  val InternalServerError: ErrorCode     = ErrorCode("INTERNAL_SERVER_ERROR", INTERNAL_SERVER_ERROR)
+  val SchemaValidation: ErrorCode        = ErrorCode("SCHEMA_VALIDATION", OK)
+  val UnsupportedMediaType: ErrorCode    = ErrorCode("UNSUPPORTED_MEDIA_TYPE", UNSUPPORTED_MEDIA_TYPE)
+  val BusinessValidationError: ErrorCode = ErrorCode("BUSINESS_VALIDATION_ERROR", BAD_REQUEST)
```

Comment:
This should also be an `OK`, as the input was totally fine, it's just not been validated - same as the schema validation.

I should check to see what `common-transit-convention-traders` expects, now I think about it...

Reply:
currently, `common-transit-convention-traders` will pass the  same 400 back to the trader.  As per the ticket we should return 400

Reply:
I raised a ticket for this. We'll not worry about it here.